README.md

```diff
@@ -210,7 +211,7 @@ Can be found at: `example/deploy/crds/mellanox.com_v1alpha1_macvlannetwork_cr.ya
 * RDMA capable hardware: Mellanox ConnectX-4 NIC or newer.
 * NVIDIA GPU and driver supporting GPUDirect e.g Quadro RTX 6000/8000 or Tesla T4 or Tesla V100 or Tesla V100.
 (GPU-Direct only)
-* Operating System: Ubuntu18.04 with kernel `4.15.0-109-generic`.
```

Comment:
but we support Ubuntu 18.04 in the alpha so why to remove it?

Reply:
will keep 18.04, its not clear whether we need to keep support for it or not imo as egx stack is switching to 20.04